### DISCUSSION TITLE
How to modify the prompt text of the restore button in the toolbar?

### DISCUSSION THREAD

**Discussion by piterzhou** [eclipse-theia/theia#14068] [Q&A]
How to modify the prompt text of the restore button in the toolbar?
I want to change the tooltip text from CTRL+SHIFT+Z to CTRL+Y

**Comment by msujew** [comment]
Hey @piterzhou,

The tooltip is generated dynamically from the keybindings. You can adjust them using a keybinding contribution. See [our documentation](https://theia-ide.org/docs/commands_keybindings/).

**Comment by piterzhou** [comment]
Thank you very much. I solved this problem by modifying the key binding. Thank you again
